lib/suspenders/adapters/heroku.rb

```diff
@@ -34,6 +33,46 @@ def set_heroku_rails_secrets
         end
       end
 
+      def provide_review_apps_setup_script
+        app_builder.template(
+          "bin_setup_review_app.erb",
+          "bin/setup_review_app",
+          force: true,
+        )
+        app_builder.run "chmod a+x bin/setup_review_app"
+      end
+
+      def create_heroku_pipelines_config_file
+        template "app.json.erb", "app.json"
+      end
+
+      def create_heroku_pipeline
+        heroku_app_name = app_name.dasherize
+
+        pipelines_plugin = `heroku plugins | grep pipelines`
+        if pipelines_plugin.empty?
+          puts 'You need heroku pipelines plugin. Run: heroku plugins:install heroku-pipelines'
+          exit 1
+        end
+        app_builder.run_heroku(
+          "pipelines:create #{heroku_app_name} -a #{heroku_app_name}-staging --stage staging",
+          "staging",
+        )
+        app_builder.run_heroku(
+          "pipelines:add #{heroku_app_name} -a #{heroku_app_name}-production --stage production",
+          "production",
+        )
+      end
+
+      def set_heroku_serve_static_files
+        %w(staging production).each do |environment|
+          app_builder.run_heroku(
+            "config:add RAILS_SERVE_STATIC_FILES=true",
```

Comment:
> Could this be resolved by setting this in `config/application.rb` instead? Less environment variables to think of.

I think that's a good idea. This is moved code though, I'd do it as part of another PR. For reference, it was introduced in
https://github.com/thoughtbot/suspenders/commit/3f937cdb3c4e945b85809e2e6be48f8af59a910f. Thanks!